docs/content/guides/non-argument-deps.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: Assets without I/O | Dagster Docs
+description: Often we don't want to build or load our assets as Python objects in memory
+---
+
+# Assets without I/O
+
+In the tutorial, all the assets we've looked at are straightforward to work with as Python objects in memory - e.g. `topstories` is created by constructing a DataFrame of the top stories from the Hacker News API.
+
+However, often we don't want to build or load our assets as Python objects in memory. For example, we might compute an asset by copying a file, by issuing a `create table` statement to a database, or by executing a command-line utility.
+
+In those situations, we can define assets whose functions don't have return values or arguments. And we can mix those assets with assets that do have return values or arguments.
+
+---
+
+## Prerequisites
+
+This guide builds off of the project written in the [tutorial](/tutorial). If you haven't already, you should complete the tutorial before continuing.
+
+---
+
+## Writing an asset with no output
+
+In the [tutorial](/tutorial), we defined an asset that contains a list of stopwords to filter out common words or prepositions from our word cloud. The English language has a lot of stopwords, so rather than hardcoding them into your asset, you'll bring them in from a separate data source.
+
+You'll augment your asset graph with a new asset that contains the stopwords from a zip file on the internet and create an asset from those stopwords. These stopwords will replace the hardcoded stopwords in your `topstories_word_cloud` asset. By the end of this guide, your asset graph will have two new assets:
+
+- `stopwords_zip` - A zip file on our local filesystem, which contains the stopwords.
+- `stopwords_csv` - A list of stopwords, which contains the unzipped contents of `stopwords_zip`.
+
+### A zip file of stopwords from the internet
+
+Let's start with `stopwords_zip`. We'll add this to the same `assets.py` you wrote in the [tutorial](/tutorial):
+
+```python
```

Comment:
This is currently failing because the file doesn't exist on `docs.dagster.io` yet, so I don't have a good way to test this. :3